docs/integrations/destinations/duckdb.md

```diff
@@ -1,16 +1,28 @@
 # DuckDB
 
-:::danger
+:::caution
 
-This destination is meant to be used on a local workstation and won't work on Kubernetes
+This destination is meant to be used on a local workstation or with the MotherDuck service as the destination. Local file-based DBs will not work on Kubernetes or in Airbyte Cloud.
```

Comment:
This connector is not enabled in cloud - if I understand correctly that should be part of the PR; if that's the case the `enabled` flag in the metadata.yaml has to be adjusted

Reply:
Thanks for calling this out. And yes, we definitely want to enable for Cloud as part of this PR. 👍